create-container.sh

```diff
@@ -9,6 +9,10 @@ set -e
 # shellcheck source=/dev/null
 source "$PWD/.devenv"
 
+# Defaults
+project_uid=1000
+project_gid=1000
```

Comment:
This probably works only if the user you are using to create the new  development environment has these `uid` and `gid`. Why don't we ask for the `uid` and `gid` of the user running the script?